content/en/infrastructure/cloud_cost_management.md

```diff
@@ -10,29 +10,44 @@ kind: documentation
 Datadog ingests your cloud cost data and transforms it into queryable metrics. If costs rise, you can correlate the change with usage metrics to determine the root cause.
  
-## Setup
- 
-To use Cloud Cost Management, you must have an AWS account with access to Cost Usage Reports (CURs), and have the AWS integration installed in Datadog.
+To use Cloud Cost Management, you must have an AWS account with access to Cost and Usage Reports (CURs), and have the AWS integration installed in Datadog.
  
-**Note:** Data can take up to 48 to 72 hours after setup to stabilize in Datadog.
+## Setup
+
+To setup Cloud Cost Management in Datadog, you need to generate a Cost 
  
-### Generate the CUR
+### Prerequisite: Generate a Cost and Usage Report
  
-Follow AWS instructions for [Creating Cost and Usage Reports][1], and select the following Content options:
+Follow AWS instructions for [Creating Cost and Usage Reports][1], and select the following Content options for use with Datadog Cloud Cost Management:
  
 * **Include resource IDs**
-* **Automatically refresh your Cost & Usage report**
+* **Automatically refresh your Cost & Usage Report**
  
 Select the following Delivery options:
  
 * Time granularity: **Hourly**
 * Report versioning: **Create new report version**
 * Compression type: **GZIP**
 * Format: `text/csv`
- 
-### Configure access to the CUR
- 
-Configure AWS to ensure Datadog has permissions to access the CUR and the s3 bucket it is stored in by [creating a policy][2] using the following JSON:
+
+### Configure the AWS integration
+
+Select your AWS billing account from the dropdown. Once selected, Datadog displays any associated tags with that account. If you have many similarly-named billing accounts, viewing the tags associated with a selected account can help ensure that you've selected the specific one you want.
+
+### Locate the Cost and Usage Report
+
+If you have navigated away from the report that you created in the setup prerequisites section, follow AWS documentation to find and [view your Cost and Usage Reports details][2].
+
+To get Datadog to locate the Cost and Usage Report, complete the fields with their corresponding details:
+
+* **Region**: This is the region your bucket is located. For example, `us-east-1`.
+* **Bucket Name**: This is the name of the s3 bucket that the CUR is saved to.
+* **Report Path Prefix**: This is the folder name. If viewing **Report path prefix** from the AWS details page, this is the first section of the path. For example, if **Report path prefix** is displayed as `cur-report-dir/cost-report`, you would enter `cur-report-dir`.
+* **Report Name**: This is the name you entered when you generated the report in the prerequisite section. If viewing the **Report path prefix** from the AWS details page, this is the second half of the path. For example, if **Report path prefix** is displayed as `cur-report-dir/cost-report`, you would enter `cost-report`.
+
+### Configure access to the Cost and Usage Report
```

Comment:
This is UI text, so I followed that rather than keeping the heading sentence-case